api/src/main/java/org/codice/ddf/admin/api/handler/report/ProbeReport.java

```diff
@@ -74,8 +74,18 @@ public ProbeReport probeResults(Map<String, Object> results) {
         return probeResults;
     }
 
+    public <T> T getProbeResult(String key) {
+        if(probeResults.get(key) != null) {
```

Comment:
`if X does not equal null, then return X; otherwise, return null` is the same as `return X`

It is also awkward that on line 78 you're using the protected field but on line 79 you're using the getter with the same name.